src/dr_wandb/wandb_run.py

```
@@ -94,13 +100,17 @@ def history_keys_last_step(self) -> int | None:
+    """Pair one stored run payload with the time it was exported."""
+
     run: WandbRun
     exported_at: str
 
     @property
     def sort_key(self) -> tuple[str, str]:
+        """Return the newest-first sort key used for persisted snapshots."""
         return (self.run.created_at or "", self.run.run_id)
 
 
 def _optional_str(value: Any) -> str | None:
```

Ruff (0.15.9)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `value`

(ANN401)